app/views/customers/show.html.erb

```diff
@@ -15,10 +15,11 @@
 		</dd>
 
 	<dt>Requirements</dt>
-	<dd><%= @customer.require_background_checks? ? "Background checks<br>".html_safe : "" %>
-		<%= @customer.require_risk_form? ? "Mentor consent<br>".html_safe : "" %>
-		</dd>
-
+	<dd><%= @customer.require_background_checks? ? "Background checks<br>".html_safe : "Background Checks "+@customer.helper_background_check_validity_length+", "%>
+		<%= @customer.require_driver_form? ? "Driver Validity<br>".html_safe : "Driver Trainings "+@customer.helper_driver_training_validity_length+", "  %>
+		<%= @customer.require_risk_form? ? "Mentor consent<br>".html_safe : "Mentor Consent, " %>
```

Comment:
I think this line and line 21 could display a blank string if the risk or conduct form isn't required.